usage-examples/editorial-commands.ly

```diff
@@ -0,0 +1,34 @@
+\version "2.19.42"
+
+\include "scholarly/package.ly"
+
+\setOption scholarly.editorial.addition #`(
```

Comment:
Okay I can see what you mean better now. Directory-wise, this `editorial-functions` module (or `editions`?) would be separate from the annotate directory, right?